tests/acceptance/features/apiAuthApp/token.feature

```diff
@@ -191,3 +191,81 @@ Feature: create auth-app token
     And user "Brian" has created auth-app token with expiration time "72h" using the auth-app API
     When user "Admin" tries to delete the last created auth-app token using the auth-app API
     Then the HTTP status code should be "403"
+
+
+  Scenario: admin user creates user using auth-app token
+    And user "Admin" has created auth-app token with expiration time "1h" using the auth-app API
+    When the administrator creates user "Brian" using the auth-app token
+    Then the HTTP status code should be "201"
+
+
+  Scenario: user lists their drives using auth-app token
+    Given user "Alice" has created auth-app token with expiration time "1h" using the auth-app API
+    When user "Alice" lists all her drives using the auth-app token
+    Then the HTTP status code should be "200"
+    And the JSON response should contain space called "Alice Hansen" and match
+      """
+      {
+        "type": "object",
+        "required": [
+          "driveType",
+          "driveAlias",
+          "name",
+          "id",
+          "quota",
+          "root",
+          "webUrl"
+        ],
+        "properties": {
+          "name": { "const": "Alice Hansen" },
+          "driveType": { "const": "personal" },
+          "driveAlias": { "const": "personal/alice" },
+          "id": { "pattern": "%space_id_pattern%" },
+          "quota": {
+             "type": "object",
+             "required": ["state"],
+             "properties": {
+                "state": { "const": "normal" }
+             }
+          },
+          "root": {
+            "type": "object",
+            "required": ["webDavUrl"],
+            "properties": {
+                "webDavUrl": { "const": "%base_url%/dav/spaces/%space_id%" }
+             }
+          },
+          "webUrl": {
+            "const": "%base_url%/f/%space_id%"
+          }
+        }
+      }
+      """
+
+
+  Scenario: admin tries to access resource of another user using auth-app token
+    Given user "Alice" has created auth-app token with expiration time "72h" using the auth-app API
+    When user "Admin" tries to list the drives of user "Alice" using the auth-app token
+    Then the HTTP status code should be "401"
```

Comment:
Is this scenario valid? If you are using alice user auth-app token, then you should use alice name rather than your own